moveit_ros/planning_interface/test/python_move_group.py

```diff
@@ -56,6 +56,22 @@ def test_validation(self):
         plan3 = self.plan(current)
         self.assertTrue(self.group.execute(plan3))
 
+    def test_get_jacobian_matrix(self):
+        current = self.group.get_current_joint_values()
+        matrix = self.group.get_jacobian_matrix(current)
```

Comment:
Please rename `matrix` -> `result` and `answer` -> `expected`.

Reply:
Thanks, I changed them.